packages/reports/tests/acceptance/report-test.js

```diff
@@ -191,7 +177,8 @@ module('Acceptance | Navi Report', function(hooks) {
     );
 
     // Remove a metric
-    await clickItem('timeGrain', 'Week');
+    await click('.navi-column-config-item__trigger');
```

Comment:
is that comment correct?

Reply:
Yes, the select choose on line 181 removes the metric

Reply:
?

Reply:
wait no that would add the Week dimension again, sorry let me fix that.